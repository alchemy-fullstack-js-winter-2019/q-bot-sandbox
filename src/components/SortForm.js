import React from 'react';

export default function SortForm({ value, handleChange}) {
  return (
    <select onChange={handleChange}>
      <option name="desc" value="desc">Descending</option>
      <option name="asc" value="asc">Ascending</option>
    </select>
  )
}
