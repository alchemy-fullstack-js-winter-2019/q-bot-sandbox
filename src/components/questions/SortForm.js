import React from 'react';

export default function SortForm({ handleChange }) {
  return (
    <>
      <label className={'sortLabel'}>Sort By Timestamp:</label>
      <select onChange={handleChange}>
        <option name="desc" value="desc">Descending</option>
        <option name="asc" value="asc">Ascending</option>
      </select>
    </>
  )
}
