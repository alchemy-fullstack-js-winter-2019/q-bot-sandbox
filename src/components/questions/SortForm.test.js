import React from 'react';
import { shallow } from 'enzyme';
import SortForm from './SortForm';

describe('SortForm', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <SortForm handleChange={jest.fn()}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
