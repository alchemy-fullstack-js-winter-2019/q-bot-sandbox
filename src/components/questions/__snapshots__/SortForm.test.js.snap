// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortForm matches a snapshot 1`] = `
<Fragment>
  <label
    className="sortLabel"
  >
    Sort By Time:
  </label>
  <select
    onChange={[MockFunction]}
  >
    <option
      name="desc"
      value="desc"
    >
      Descending
    </option>
    <option
      name="asc"
      value="asc"
    >
      Ascending
    </option>
  </select>
</Fragment>
`;
